.travis.yml

```diff
@@ -6,7 +6,7 @@
 # - sudo/dist/group are set so as to get Blue Box VMs, necessary for [loopback]
 #   IPv6 support
 
-sudo: false
+sudo: required
```

Comment:
Ah, that's the background of Bitcoin devs using this setup - need to dig further into this, i am not biased to a specific one though